Populate ack (#470)
usegalaxy-eu/website: about.md

<br>
### ACKNOWLEDGEMENT

  We are aiming to maintain high competency and provide high-quality data analysis services to all our Galaxy users.
  
  Therefore, we request that you acknowledge this service by including the members of the Freiburg Galaxy Team as co-authors if they have made a significant intellectual and/or organizational contribution to the work described (conceptualization, design, data analysis, data interpretation and/or input into drafting, revising or writing any portion of the manuscript).

  Individuals who have contributed to the project, but whose contributions do not rise to the level justifying authorship, can be recognized in the acknowledgements section of the manuscript as follows:

  > The authors acknowledge the support of the Freiburg Galaxy Team: Person *X* and Prof. Rolf Backofen, Bioinformatics, University of Freiburg (Germany) funded by the [Collaborative Research Centre 992 Medical Epigenetics](http://www.sfb992.uni-freiburg.de) ([DFG](http://www.dfg.de) grant SFB 992/1 2012) and the German Federal Ministry of Education and Research [BMBF](http://www.bmbf.de/) grant 031 A538A [de.NBI](https://www.denbi.de)-RBC.

>   Additional funding of projects and/or provision of material expenses are welcome as well, to help support our growing Galaxy community in Freiburg and Europe.

Author: ```suggestion
  Additional funding of projects as well as the provision of material expenses is welcome to support our growing Galaxy community in Freiburg and Europe.
```
